src/Apps/Settings/Routes/EditProfile/Mutations/useVerifyID.ts

```diff
@@ -0,0 +1,31 @@
+import { graphql } from "react-relay"
+import { useMutation } from "Utils/Hooks/useMutation"
+import { useVerifyIDMutation } from "__generated__/useVerifyIDMutation.graphql"
+
+export const useVerifyID = () => {
+  return useMutation<useVerifyIDMutation>({
+    mutation: graphql`
+      mutation useVerifyIDMutation(
+        $input: SendIdentityVerificationEmailMutationInput!
+      ) {
+        sendIdentityVerificationEmail(input: $input) {
+          confirmationOrError {
+            ... on IdentityVerificationEmailMutationSuccessType {
+              identityVerification {
```

Comment:
Note that this refactor changed the mutation to use `identityVerification` field and _not_ `identityVerificationEmail`, which [has been deprecated](https://github.com/artsy/metaphysics/blob/bf6ff5cd64c9dae0e46d859657fb016e514984ac/src/schema/v2/me/sendIdentityVerficationEmailMutation.ts#L56-L60).